src/rpc/Factories.cpp

```diff
@@ -85,13 +85,13 @@ makeHttpContext(
 )
 {
     if (!request.contains("method"))
-        return Error{{ClioError::RpcCommandIsMissing}};
+        return Error{{ClioError::RpcCommandIsMissing, "Method is not specified."}};
```

Comment:
The messages are already in `Errors.cpp` and should not need to be added here or below

Reply:
@emreariyurek do you still want to fix it?